web/src/admin/NewEndpoint.vue

```diff
@@ -12,7 +12,7 @@
     <div class="p-3 bg-red-100 rounded-md" v-if="formErrors.length">
       <b>Please correct the following error(s):</b>
       <ul>
-        <li v-for="error in formErrors">* {{ error }}</li>
+        <li v-for="(error, idx) in formErrors" :key="`err-${idx}`">* {{ error }}</li>
```

Comment:
Nitpick: the `key` could just be `idx` as it only needs to be unique withint this `v-for` statement.